src/pdsfile/holdings_maintenance/pds3/shelf_consistency_check.py

```
@@ -23,65 +24,67 @@
 
-            if verbose:
-                print(holdings_path)
+            # Convert to the associated holdings path
+            holdings_path = shelf_path.replace('shelves/' + tail, 'holdings')
+            holdings_path = holdings_path.rpartition('.')[0]
+
+            # For index shelves, make sure the holdings label file exists
+            if tail == 'index':
+                if not os.path.exists(holdings_path + '.lbl'):
+                    print('*** Extraneous shelf: ' + shelf_path)
+                    error += 1
```

Ruff (0.16.1)

[error] Undefined name `error`

(F821)